homeassistant/components/ihc/ihcdevice.py

```diff
@@ -0,0 +1,40 @@
+"""Implements a base class for all IHC devices."""
+
+
+class IHCDevice:
+    """Base class for all ihc devices."""
+
+    def __init__(self, ihccontroller, name, ihcid, ihcname: str,
```

Comment:
Please use snake case variable names when the names contain multiple words for legibility. Eg:
```py
def __init__(self, ihc_controller, name, ihc_id, ihc_name: str,
```

Reply:
Done